integration_tests/CMakeLists.txt

```diff
@@ -331,6 +331,7 @@ RUN(NAME test_builtin_divmod LABELS cpython llvm c)
 RUN(NAME test_builtin_sum    LABELS cpython llvm c)
 RUN(NAME test_math1          LABELS cpython llvm c)
 RUN(NAME test_math_02        LABELS cpython llvm)
+RUN(NAME test_math_03        LABELS llvm)
```

Comment:
Add a TODO that this needs to be tested with Python 3.11 in future.

Reply:
I already added a todo as new issue associated with this PR.

Reply:
I think Gagan is saying to add something like this:
https://github.com/lcompilers/lpython/blob/37918fa535b587040cf2ee91e37f402b76360e74/integration_tests/CMakeLists.txt#L435

Reply:
Okay. Got it.

Reply:
Added.